packages/common/src/models/Analytics.ts

```diff
@@ -2530,6 +2535,7 @@ export type AllTrackingEvents =
   | PlaybackPlay
   | PlaybackPause
   | BufferingTime
+  | BufferSpinnerShown
```

Comment:
I'm a little worried about the number of events we will send to amplitude. just a lot of network volume when you click play on something. fine for a little while, but i think we should consider removing these metrics at some point or consolidating it all into one "Play" event with properties

Reply:
Fair point. I looked into a bit and its a bit messy to aggregate all the calls with the data points we would want for a single play event. My main goal for this is just to evaluate if performance improvements are hitting the mark or not, so if you're okay with it I'm down to use this just the week or so after launch and delete after